nulib/digital-collections
# Add privacy link to app Footer.  Remove node-sass dependency in app s…

…o it will run in modern versions of NPM

## Summary 
Adds a simple Privacy link to the app footer.  Also removes the `node-sass-chokidar` dependency (`node-sass`) which is deprecated and doesn't run in modern versions of `npm`. 

## Specific Changes in this PR
- Privacy link added
- `node-sass-chokidar` replaced with `sass`

## Steps to Test
1. Wipe out your `node_modules` folder and `package-lock.json`.
2. `npm install`
3. Fire up the app.  Notice that it builds.
4. Verify Privacy Policy link appears

Also please let developers know if there are any special instructions to test this in the development environment.
